sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -304,6 +305,10 @@ public void testUpdateStmtBuilder() {
         printStatement("update foo set col['x'] = 3 where foo['x'] = 2");
         printStatement("update schemah.foo set foo.a='b', foo.b=foo.a");
         printStatement("update schemah.foo set foo.a=abs(-6.3334), x=true where x=false");
+        printStatement("update foo set foo='a' returning id");
```

Comment:
can we maybe have separate test cases for the returning clause?

Reply:
👍